src/main/java/com/chatapp/vartalaab/service/UserSessionService.java

```diff
@@ -24,4 +25,25 @@ public void saveUserSessionDetails(UserSessionDetails userSessionDetails){
     public Optional<UserSessionDetails> getUserSessionDetails(String username){
         return userSessionRepository.findById(username  + GeneralUtility.SESSION_APPENDER);
     }
+
+    public void updateUserWebSocketSessionDetails(String username, String sessionId, boolean shouldAdd){
+        Optional<UserSessionDetails> userSession = this.getUserSessionDetails(username);
+        if(shouldAdd){
```

Comment:
instead of shouldAdd, can you use some different name for ex :-> isOffline/isOnline, or if you think of something that adds more semantic meaning to it